packages/core/src/lib/table/table.svelte

```diff
@@ -0,0 +1,60 @@
+<!--
+@component
+  
+A table.
+```svelte
+<Table variant='fixed' cols=["30%", "30%", "40%"]>
```

Comment:
How does the fixed table behave when contents overflow a cell?

Reply:
it wraps to maintain the widths, this example is with widths 10% and 90%
![image](https://github.com/viamrobotics/prime/assets/31592018/c5e30a00-3046-4fe3-88a3-bab6fc960805)